services-worker/src/test/resources/edu/unc/lib/dl/cdr/services/techmd/fitsOutputMultipleLineBreaks.xml

```diff
@@ -1,5 +1,3 @@
-2015-06-13 06:19:59,410 ERROR [main] ToolBelt:79 - Thread 1 error initializing edu.harvard.hul.ois.fits.tools.exiftool.Exiftool: edu.harvard.hul.ois.fits.exceptions.FitsToolCLIException  Message: Error calling external command line routine (Cannot run program "which": java.io.IOException: error=12, Cannot allocate memory)
```

Comment:
I think we should retain this line while the test that relies on it still exists https://github.com/UNC-Libraries/Carolina-Digital-Repository/blob/a7f3c69d0ab4fe71d3a013b3874a47d13d198ab8/services-worker/src/test/java/edu/unc/lib/dl/cdr/services/techmd/TechnicalMetadataEnhancementTest.java#L75

This was here to make sure that the TechMD enhancement could handle dirty output from FITS, which unfortunately has historically been a thing.  The TechMD service class will need to be replaced, since it deals with FOXML, but the part that interacts with the output from FITS will likely need to be the same.